codestodians/komlie-frontend
# [Chore] Create utils, update existing

#### What does this PR do?
- Replaces the encryption module
- Introduces new utils to handle `localStorage` operations
- Refactors for the setting of `komlie` items in localStorage with a predefined prefix

#### How should this be manually tested?
- Login/Signup/Logout and see that new items set in local storage for the app are prefixed with `kom`
